core-banking-bootstrap.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+print_usage() {
+  echo "[INFO]        Usage:"
+  echo "[INFO]              ./core_banking_bootstrap.sh [skipBuild]"
+  echo "[INFO]              skipBuild: optional flag to skip images build"
+  echo "[INFO]        Note: 1) must be run from the project root dir"
+  echo "[INFO]              2) docker must be running"
+}
+
+if [[ -n "$1" && "$1" != "skipBuild" ]]; then
+  echo "[ERROR] Unknown flag $1."
+  print_usage
+  exit 1
+fi
+
+echo "[INFO] Checking if docker daemon is running..."
+rep=$(curl -s --unix-socket /var/run/docker.sock http://ping >/dev/null)
+status=$?
+if [ "$status" == "7" ]; then
+  echo '[ERROR] docker is not running, please start docker.'
+  print_usage
+  exit 1
+fi
+
+echo "[INFO] Checking the current path..."
+if [[ $0  != "./core-banking-bootstrap.sh" ]]; then
+  echo "[ERROR] Current path is $(pwd), please run from the project root dir .../../core-banking."
```

Comment:
Thx for changing this, could you please also update the printed error message, it still says `.../../core-banking`. I think saying "please run from the project root dir" should be enough. Having the `.../../core-banking` is a bit confusing since we cannot be really sure what the current directory is.